openpaas-james/integration-tests/jmap/distributed-jmap-integration-tests/pom.xml

```diff
@@ -29,6 +29,12 @@
             <artifactId>jmap-integration-tests-common</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.linagora.openpaas</groupId>
+            <artifactId>mailbox-opendistro-tests</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
         <dependency>
```

Comment:
Don't forget to configure surefire tests for distributed server, so that we do not need to run them all!

```
    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <configuration>
                    <argLine>-Xms512m -Xmx2048m</argLine>
                    <reuseForks>true</reuseForks>
                    <!-- Fail tests longer than 20 minutes, prevent form random locking tests -->
                    <forkedProcessTimeoutInSeconds>1200</forkedProcessTimeoutInSeconds>
                    <!-- Junit 5 move from Category to Tag, and it need tag's name as parameter -->
                    <groups combine.self="override">BasicFeature</groups>
                    <excludedGroups>unstable</excludedGroups>
                    <properties>
                        <includeTags>junit5</includeTags>
                    </properties>
                </configuration>
            </plugin>
        </plugins>
    </build>
```